docs/detectors/vcpkg.md

```diff
@@ -0,0 +1,21 @@
+# vcpkg Detection
+
+## Requirements
+
+vcpkg detection triggers off of `vcpkg.spdx.json` files found under the scan directory.
```

Comment:
Most folks are completely unaware of `vcpkg.spdx.json` files, can we add a note that it is required for them to use VCPKG to build in their system and use <X> version which will generate those .spdx.json files.